packages/frontend/src/components/DashboardTiles/TileForms/AddChartTilesModal.tsx

```diff
@@ -95,7 +95,7 @@ const AddChartTilesModal: FC<Props> = ({ onAddTiles, onClose }) => {
                 return {
                     uuid: uuid4(),
                     properties: {
-                        title: savedChart?.name || null,
+                        title: savedChart?.name,
```

Comment:
Do we want to save the chart title as the tile tile when the tile is created? 
If I change the chart title it won't be reflated in the dashboard. 

I thought we were going to have that input optional as show the chart name as a placeholder. so they now it defaults to the chart name. @PriPatel

Reply:
@ZeRego 
If I don't edit the tile title:
- The default tile title should be the saved chart's name
- If I edit the saved chart's name , it should update the tile's name name on the dashboard.

If I edit the tile title:
- If I edit the tile title, any changes to the saved chart name should not affect the tile.